arbitrum-docs/build-decentralized-apps/how-to-use-timeboost.mdx

```diff
@@ -0,0 +1,473 @@
+
+We can use this event to determine whether or not we've won the auction. The event signature is:
+
+```solidity
+event SetExpressLaneController(
+    uint64 round,
+    address indexed previousExpressLaneController,
+    address indexed newExpressLaneController,
+    address indexed transferor,
+    uint64 startTimestamp,
+    uint64 endTimestamp
+);
+```
+
+Here's an example to get the log from the auction contract to determine the new express lane controller:
+
+```tsx
+const fromBlock = <any recent block, for example during the auction>
+const logs = await publicClient.getLogs({
+  address: auctionContractAddress,
+  event: auctionContractAbi.filter((abiEntry) => abiEntry.name === 'SetExpressLaneController')[0],
+  fromBlock,
+});
+
+const newExpressLaneController = logs[0].args.newExpressLaneController;
+console.log(`New express lane controller: ${newExpressLaneController}`);
+```
+
+If you won the auction, congratulations! You are the express lane controller for the next round, which, by default, will start 15 seconds after the auction closed. The following section explains how we can submit a transaction to the express lane.
+
+## How to submit transactions to the express lane
+
+Transactions that are sent to the express lane are immediately sequenced by the sequencer, while regular transactions are delayed 200ms by default. However, only the express lane controller can send transactions to the express lane. The previous section explained how to participate in the auction to be the express lane controller for a given round.
+
+The express lane is handled by the sequencer, so transactions are sent to the sequencer endpoint. We need to send a `timeboost_sendExpressLaneTransaction` request with the following information:
+
+- chain id
+- current round (following the example above, `currentRound`)
+- address of the auction contract
+- sequence number: a per-round nonce of express lane submissions, which is reset to 0 at the beginning of each round
+- RLP encoded transaction payload
+- conditional options for Arbitrum transactions ([more information](https://github.com/OffchainLabs/go-ethereum/blob/48de2030c7a6fa8689bc0a0212ebca2a0c73e3ad/arbitrum_types/txoptions.go#L71))
+- signature (explained below)
+
+::::info Timeboost-ing third party transactions
+
+Notice that, while the express lane controller needs to sign the `timeboost_sendExpressLaneTransaction` request, the actual transaction to be executed can be signed by any party. In other words, the express lane controller can receive transactions signed by other parties and sign them to apply the time advantage offered by the express lane to those transactions.
```

Comment:
The second bit of the correction might lead to some misconceptions, so I prefer to be a bit more clear with it. For context, what's happening here is that the express lane controller (ELC) will sign and send the request to the sequencer, but within that request there's a transaction signed by whoever wishes their transaction to be sent through the express lane (might be the ELC or a third party). That signer must collaborate with the ELC though, since the ELC is the one sending the transaction to the sequencer.

I'm not completely happy with my description either, but the correction might lead to thinking that anyone (regardless of them collaborating with the ELC) can sign the transaction.

I'll keep the original text (with your first correction), but happy to hear other alternatives:

`Notice that while the express lane controller must sign the `timeboost_sendExpressLaneTransaction` request, the actual transaction to be executed can be signed by any party. In other words, the express lane controller can receive transactions signed by other parties and sign them to apply the time advantage offered by the express lane to those transactions.`